config/nginx.conf.template

```diff
@@ -231,6 +231,13 @@ http {
       proxy_pass http://portainer:9000/;
     }
 
+ # Redirect traffic from /deploy/* to the autodeployment server
+ # extra slash for accepting all request_uri e.g. /deploy/status
```

Comment:
Can you bump these comments in so they align with the location on 236?

Reply:
Do you mean just put them on the same line? I'm not familiar with "bump"

Reply:
Sorry "bump them in one tab stop":

```
BEFORE
# Redirect traffic from /deploy/* to the autodeployment server
# extra slash for accepting all request_uri e.g. /deploy/status
    location...


AFTER
    # Redirect traffic from /deploy/* to the autodeployment server  
    # extra slash for accepting all request_uri e.g. /deploy/status
    location...
```